teachertool/src/components/styling/CatalogOverlay.module.scss

```diff
@@ -77,7 +90,7 @@
                     .action-indicators {
                         position: relative;
                         padding: 0 1rem 0 0;
-                        width: 3rem;
+                        width: 3rem; // Match common-accordion-chevron
```

Comment:
Is this comment needed?

Reply:
I think it's a nice reminder if someone ever changes it, they should consider changing it there too (which may or may not look good depending on the new value, up to the engineer at that point). Could go and do a whole var for it but that seemed like overkill to me.

Reply:
Hmm, I'm not sure if that warrants a comment, personally. I feel like there is a lot of styling that is made in order to match with styling elsewhere, and leaving that up to experimentation is fine.

Reply:
If I were the engineer making the change, I would appreciate the comment because it's the kind of thing I'd otherwise miss :) But if you feel very strongly I can remove it.

Reply:
Nope, don't feel strongly; fine to keep it in.